lib/ImageWorker.js

```diff
@@ -34,33 +29,71 @@ const wrappedComponent = WrappedComponent => props => {
   return <WrappedComponent {...props} />;
 };
 
+/** Have we initiated the worker pool already? */
+const workerPoolCreated = false;
```

Comment:
I think you will have to change this to a `let` and reassign it to a true, after the `createWorkerPool` fn is invoked. Else everytime there will be 4 workers created as `workerPoolCreated` will always be false